user-guide/scheduler.rst

```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
 Resource Limits
 ---------------
 
-There are different resource limits on ARCHER2 for different purposes.
+The ARCHER2 resource limits for any given job are covered by three separate attributes.
+
+* The amount of *primary resource* you require (i.e., CPU cores).
+* The *partition* that you want to use - this specifies the nodes that are eligible to run your job.
+* The *Quality of Service (QoS)* that you want to use - this specifies the job limits that apply.
+
+Each of these attributes is described in more detail below. 
+
+The *primary resources* you request are *compute* resources: CPU cores on the compute nodes. Other node resources,
+such as memory, are assigned pro rata based on the primary resource that you request.
+
+.. warning::
+
+   On ARCHER2, you cannot specify the memory for a job using the ``--mem`` options to Slurm
+   (e.g. ``--mem``, ``--mem-per-cpu``, ``--mem-per-gpu``). The amount of memory you are 
+   assigned is calculated from the amount of primary resource you request.
```

Comment:
As "--exclusive" is enforced on ARCHER2, there is no pro-rata calculation of memory - you get access to all of the memory per node, no matter what distribution of processes/threads you ask for.

Reply:
OK